doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -323,6 +323,10 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0
+
+KRKNWK-18556: While creating multiple subscriptions by the multiple Matter controllers, the persistent subscriptions resumption feature works only for the first created subscriptions.
```

Comment:
```suggestion
KRKNWK-18556: While creating multiple subscriptions, the persistent subscriptions resumption feature works only for the first created subscriptions
  This happens when multiple subscriptions are created by multiple Matter controllers.
```
The rendering of the known issues page requires two lines of content. If there some other information that could be added instead, that is also fine instead of splitting the current information in two.

Reply:
Thanks for the explanation :) I fixed that and added one more line.

Reply:
I have now added this known issue to https://github.com/nrfconnect/sdk-nrf/pull/14185.